### PR TITLE
[RISK=no] Cleaning up unused stuff in our build file

### PR DESCRIPTION
**Pull request by rushtong**
No functional changes - just removing some unused code.

### CODE REVIEW

**Comment by rushtong** [comment]
@calbach - PTAL, thank you.

File: api/build.gradle [hunk 1]
```diff
@@ -309,30 +306,6 @@ dependencies {
   generatedCompile 'com.fasterxml.jackson.datatype:jackson-datatype-joda:2.9.6'
 }
 
-swaggerSources {
```
Comment (calbach) [inline api/build.gradle]: Could you verify that validateSwagger still validates swagger files after this change? I'm not 100% sure, but I vaguely recall setting up some non-obvious hand-off to the swagger plugin to feed into that validation.

Comment (rushtong) [reply api/build.gradle]: Thank you, you're correct. This is required. The validation code runs, but doesn't see the files so doesn't fail. This is very non-obvious - I'll close this and look for a better way to make it clear that these are used.

Comment (calbach) [reply api/build.gradle]: Sorry, I probably should have left a big comment here explaining why we had this.

**Pull request closed without merging**